src/includes/performance/enable-tracing/android.mdx

```diff
@@ -0,0 +1,38 @@
+```xml {filename:AndroidManifest.xml}
+<application>
+  <meta-data android:name="io.sentry.traces.sample-rate" android:value="1.0" />
+</application>
+```
+
+Or, if you are manually instrumenting Sentry:
```

Comment:
Opting in to tracing with the sampling decision isn't really related to manual vs auto instrumentation (we can start capturing transactions automatically if we are able to if the users set a sampling decision in):

```suggestion
Or, if you are initializing the SDK programatically:
```

Reply:
this was the wording defined by the docs team about "auto init mode" vs "initializing the SDK by yourself".
I'd stick to it or we'd need to change everywhere, otherwise, only this page would differ.

it's not really related to tracing, but if you have the auto init mode or not

Reply:
@PeloWriter maybe has an opinion here